libs/error-analysis/src/lib/ErrorAnalysisDashboard/Controls/TreeViewRenderer/TreeViewRenderer.tsx

```diff
@@ -386,14 +331,14 @@ export class TreeViewRenderer extends React.PureComponent<
               >
                 {nodeData.map((node, index) => (
                   <CSSTransition
-                    key={node.id}
+                    key={node.id! + Math.random()}
```

Comment:
random? Is that intended?

Reply:
Yes, it's to make the ids unique, I recall otherwise the browser spits out a warning. Actually, I updated the code so the CSS transition and g have the same key.  The links have the same code.  Just to clear any confusion, the code is the same as before.